lib/config-loader.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+var cloneDeep = require('lodash').cloneDeep;
```

Comment:
I stand by what I said but maybe that's just my preference.

It shouldn't stop this PR from being landed though.